src/pages/bug-bounty/rewards.tsx

```diff
@@ -51,145 +52,158 @@ const Subtitle = styled(Header)`
-                                                weight="normal"
-                                                type="paragraph-1"
-                                                as="p"
-                                            >
-                                                {item.title_2}
-                                            </Header>
-                                            <Subtitle as="p" type="subtitle-1" align="center">
-                                                {item.sub_title_2}
-                                            </Subtitle>
-                                            <Header
-                                                align="center"
-                                                weight="normal"
-                                                type="paragraph-1"
-                                                as="p"
-                                            >
-                                                {item.title_3}
-                                            </Header>
-                                            <Subtitle as="p" type="subtitle-1" align="center">
-                                                {item.sub_title_3}
-                                            </Subtitle>
-                                        </>
-                                    )}
-                                </Card>
-                            )
-                        })}
-                    </Flex>
-                </CardWrapper>
+const rewards_text: TString = '_t_Rewards_t_'
+const bug_reports_text: TString =
+    '_t_Your bug reports are reviewed and rewarded based on impact and severity._t_'
+const see_list_text: TString = '_t_<0>See a list of the vulnerability examples</0>_t_'
 
-                <Flex ai="center" mt="16px">
-                    <Localize
-                        translate_text="<0>See a list of the vulnerability examples</0>"
-                        components={[
-                            <LocalizedLinkText
-                                mr="8px"
-                                key={0}
-                                to="/bug-bounty/vulnerabilities-types/"
-                                color="red"
-                            />,
-                        ]}
-                    />
-                    <ImageWithDireciton src={Arrow} alt="arrow" width="16" height="16" />
+const Rewards = () => (
+    <StyledSectionContainer>
+        <Container fd="column">
+            <Header as="h2" type="heading-2" align="center" mb="8px">
+                <Localize translate_text={rewards_text} />
+            </Header>
+            <Header as="p" type="paragraph-1" align="center" weight="400">
+                <Localize translate_text={bug_reports_text} />
+            </Header>
+            <CardWrapper>
+                <Flex laptopM={{ width: 'auto !important' }}>
+                    {card_content.map((item) => {
```

Comment:
```suggestion
                    {card_content.map(({ id, alt, icon }) => {
```

Reply:
Since objects structures are not the same I didn't use destructuring in here.
some objects don't have `sub_title_1` `title_2` `sub_title2` `title_3` `sub_title_3`.
We could refactor the objects and assign `null` to mentioned keys but it requries more refactoring on JSX. I decided to skip it for now since this PR is already big.